tests/conftest.py

```diff
@@ -2084,3 +2084,88 @@ def saved_hyperopt_results():
                                                                          ].total_seconds()
 
     return hyperopt_res
+
+
+@pytest.fixture(scope='function')
+def limit_buy_order_usdt_open():
+    return {
+        'id': 'mocked_limit_buy',
+        'type': 'limit',
+        'side': 'buy',
+        'symbol': 'mocked',
+        'datetime': arrow.utcnow().isoformat(),
+        'timestamp': arrow.utcnow().int_timestamp,
+        'price': 2.00,
+        'amount': 30.0,
+        'filled': 0.0,
+        'cost': 60.0,
+        'remaining': 30.0,
+        'status': 'open'
+    }
+
+
+@pytest.fixture(scope='function')
+def limit_buy_order_usdt(limit_buy_order_usdt_open):
+    order = deepcopy(limit_buy_order_usdt_open)
+    order['status'] = 'closed'
+    order['filled'] = order['amount']
+    order['remaining'] = 0.0
+    return order
+
+
+@pytest.fixture
+def limit_sell_order_usdt_open():
+    return {
+        'id': 'mocked_limit_sell',
+        'type': 'limit',
+        'side': 'sell',
+        'pair': 'mocked',
+        'datetime': arrow.utcnow().isoformat(),
+        'timestamp': arrow.utcnow().int_timestamp,
+        'price': 2.20,
+        'amount': 30.0,
+        'filled': 0.0,
+        'remaining': 30.0,
+        'status': 'open'
+    }
+
+
+@pytest.fixture
+def limit_sell_order_usdt(limit_sell_order_usdt_open):
```

Comment:
Just a thought while looking at this - (probably for the future / other PR's - as i don't see it too relevant for test_persistence) - would it make sense to have a fixture for "winning sell" and "losing sell" - so we have "standardized" orders without having to mess with the values in the tests to make it winning or losing depending on the need?

Reply:
Not really, because it's just the 1 or 2 values that need to be changed, and then everything else stays the same. The only tests where it really matters is `test_profit` and `test_profit_ratio`

Reply:
If it ends up coming up more I'll do that

Reply:
> If it ends up coming up more I'll do that

it's been a thought / idea i had while looking at this ... not sure myself if it's a neccessity ...